raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -558,7 +558,27 @@ contract TokenNetwork is Utils {
         Channel storage channel = channels[channel_identifier];
 
         require(channel.state == ChannelState.Closed);
-        // Channel must be in the settlement window
+
+        // Calling this function after the settlement window is forbidden to
```

Comment:
As discussed in rocketchat contract changes, even comments, should not go in before Red Eyes. So I am just adding a blocking review until then.